### PR TITLE
Add build script and configure for Internal CI build.

### PR DESCRIPTION
**Pull request by rghetia**
Start with Linux build. Subsequently add other platforms.

### CODE REVIEW

**Comment by sebright** [comment]
What will this build be used for?  Has Kokoro been enabled for the repository?

File: buildscripts/kokoro/linux.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# To run locally:
+#  ./buildscripts/kokoro/linux.sh
+
+# This script assumes `set -e`. Removing it may lead to undefined behavior.
+set -exu -o pipefail
+
+# It would be nicer to use 'readlink -f' here but osx does not support it.
+readonly OPENCENSUS_JAVA_DIR="$(cd "$(dirname "$0")"/../.. && pwd)"
+
+# cd to the root dir of grpc-java
```
Comment (songy23) [inline buildscripts/kokoro/linux.sh]: grpc-java -> opencensus-java

File: buildscripts/kokoro/linux.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+cd $(dirname $0)/../..
+
+# Run tests
+./gradlew clean build
+pushd examples
+./gradlew clean assemble --stacktrace
```
Comment (songy23) [inline buildscripts/kokoro/linux.sh]: Should this be `./gradlew clean assemble check --stacktrace`?

**Comment by songy23** [comment]
Also please rebase and remove the merge commit (merge commits are [disallowed](https://github.com/census-instrumentation/opencensus-java/blob/master/scripts/check-git-history.py#L19) for this repo).

File: buildscripts/kokoro/linux.sh (hunk 2 above)
Comment (rghetia) [reply buildscripts/kokoro/linux.sh]: I will fix the issues and attach the build results.

**Comment by rghetia** [comment]
[build.log](https://github.com/census-instrumentation/opencensus-java/files/1963049/build.log)

output of linux.sh is attached.

**Comment by rghetia** [comment]
@bogdandrutu can you please answer @sebright 's question of why kokoro?
@sebright , regarding your second question about enabling kokoro for this repo, I have another commit that I will shortly upload for a review.

**Pull request merged**